pom.xml

```diff
@@ -234,4 +218,18 @@
       </plugin>
     </plugins>
   </build>
+
```

Comment:
Maven central does not like additional repositories specified for dependencies: see https://maven.apache.org/repository/guide-central-repository-upload.html FAQ point 1.

That doesn't mean this is the wrong option, but it's worth some thought. I really don't want to have to create a new maven central repo and keep it in sync with buf.

Reply:
Adding the repository was a requirement for getting the generated classes, and from what I can tell all the buf plugins being used exist in the central repository so we should be fine.